brightray/brightray.gypi

```diff
@@ -271,13 +271,19 @@
               # can be removed at link time with --gc-sections.
               '-fdata-sections',
               '-ffunction-sections',
+              '-flto=thin',
             ],
             'ldflags': [
               # Specifically tell the linker to perform optimizations.
               # See http://lwn.net/Articles/192624/ .
               '-Wl,-O1',
               '-Wl,--as-needed',
               '-Wl,--gc-sections',
+              '-flto=thin',
```

Comment:
The following should be moved under `clang==1` condition, currently there is mips (experimental) that uses gcc, mips CI is not enabled on master but is available on 1-8-x branch https://github.com/electron/electron/pull/11281

Reply:
Same answer as above. Not sure if I'll be able to verify the MIPS build locally, so if anyone has it set up...

Reply:
/cc @zcbenz